.github/workflows/live-deploy.yml

```diff
@@ -0,0 +1,25 @@
+# An Action to build Web Arcs and deploy to https://live.arcs.dev
+name: live-deploy
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Is there a way to make sure only green commits on master are deployed?

Reply:
It looks like there's an action that will fit nicely here: https://github.com/marketplace/actions/wait-for-check

We can run the deploy step only after GCB passes on master.

Reply:
Do you want to make that change before merging this PR in?

Reply:
Yes, I do. 
Having read more documentation, I think a simpler / cleaner approach would be to run the typescript tests + lint in the Action.

Reply:
I am actually wondering whether this should be done as part of GCB itself to reduce the number of dependencies to other services. However, that might only delay things further. Let us start with this and move to GCB If there are issues.